apis/flowcollector/v1beta1/flowcollector_types.go

```diff
@@ -150,14 +150,16 @@ type FlowCollectorIPFIX struct {
 // Agent feature, can be one of:<br>
 // - `PacketDrop`, to track packet drops.<br>
 // - `DNSTracking`, to track specific information on DNS traffic.<br>
-// - `FlowRTT`, to track TCP latency. [Unsupported (*)].<br>
-// +kubebuilder:validation:Enum:="PacketDrop";"DNSTracking";"FlowRTT"
+// - `FlowRTT`, to track TCP latency [Unsupported (*)].<br>
+// - `NetworkEventsMonitor`, to track Network events.<br>
```

Comment:
just wondering if naming it just `NetworkEvents` would be sufficient and easier to remember, no ?

Reply:
Also, maybe add more details right here because it's not super obvious what are network events. like ".. such as network policy verdicts" ?